src/main/java/TaskManager.java

```diff
@@ -62,49 +104,85 @@ public static void addToList(String item, TaskType type) {
         Picture.printLine();
         if (itemNumber > tasksCount || itemNumber < 1) {
-            System.out.println("     NOTICE: Invalid task selected.");
+            System.out.println(ERROR_INVALID_TASK_SELECTED);
         } else {
             tasks[itemNumber - 1].markTaskAsDone();
-            System.out.println("     NOTICE: This task is marked as done...\n"
-                    + "       [X] "
-                    + tasks[itemNumber - 1].getDescription());
+            final String taskDetails = tasks[itemNumber - 1].toString();
+            System.out.println(getMessageForMarkTaskAsDone(taskDetails));
         }
         Picture.printLine();
     }
 
     /**
-     * Prints all the tasks in the list.
+     * Constructs a confirmation message when a task is marked as done.
+     *
+     * @param taskDetails String containing the description of the task.
+     * @return confirmation message for a task marked as done.
+     */
+    private static String getMessageForMarkTaskAsDone(String taskDetails) {
+        return String.format(MESSAGE_MARK_TASK_AS_DONE, taskDetails);
+    }
+
+    /**
+     * Prints an error message if an invalid task is selected;
+     * otherwise prints all the tasks in the list in ascending order.
      */
     public static void printList() {
         Picture.printLine();
         if (tasksCount == 0) {
-            System.out.println("     NOTICE: There are no tasks in your list.");
+            System.out.println(ERROR_NO_TASK_IN_LIST);
         } else {
-            System.out.println("     NOTICE: Here are the task(s) in your list...");
-            for (int i = 0; i < tasksCount; i++) {
-                System.out.println("       " + (i + 1) + "." + tasks[i].toString());
-            }
+            System.out.println(MESSAGE_TASK_IN_LIST);
+            printTasksInList();
         }
         Picture.printLine();
     }
 
     /**
-     * Prints a list of available chat-bot commands.
+     * Enumerates through all the tasks in the list and
+     * prints each task line by line.
      */
-    public static void printCommands() {
+    private static void printTasksInList() {
+        String taskDetails;
+        for (int i = 0; i < tasksCount; i++) {
+            taskDetails = tasks[i].toString();
+            System.out.println(getListItem(i + 1, taskDetails));
+        }
+    }
+
+    /**
+     * Prints one task in the list.
+     *
+     * @param taskNumber  Integer value that the task has been assigned.
+     * @param taskDetails String description of the task.
+     * @return A string numbering the task and the corresponding details.
+     */
+    private static String getListItem(int taskNumber, String taskDetails) {
+        return String.format(LIST_ITEM, taskNumber, taskDetails);
+    }
+
+    /**
+     * Prints an error message if an invalid command is used,
+     * followed by a list of possible commands.
+     */
+    public static void printInvalidCommandMessage() {
```

Comment:
Coding conventions followed well :)
Variable names are in camelCase, class names are in PascalCase
Function names are easy to understand